src/Console/Command/SelfUpdateCommand.php

```diff
@@ -151,7 +151,8 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $tempFilename = \dirname($localFilename).'/'.basename($localFilename, '.phar').'-tmp.phar';
         $remoteFilename = $this->toolInfo->getPharDownloadUri($remoteTag);
 
-        if (false === @copy($remoteFilename, $tempFilename)) {
+        copy($remoteFilename, $tempFilename);
```

Comment:
unfortunately, we should still silence the `copy` function to not get low-level warning, we are handling outcome differently.